deployer/build.rs

```diff
@@ -0,0 +1,17 @@
+use std::{env, process::Command};
+
+fn main() {
+    println!("cargo:rerun-if-changed=runtime");
```

Comment:
I tried changing this to an absolute path to runtime now (`/home/oddgrd/dev/shuttle/runtime`), and then it doesn't recompile on every `cargo shuttle run` anymore.